elastic/kibana
# Add circuit breaker for max number of actions by connector type

fixes: #126504

## Summary
This PR intends to introduce a new kibana.yml config key that can override the actions.max circuit breaker by connector type(s) 

## To Verify
Create a rule that has more than one actions of the same type (e.g. Server Log).
Add the below config to your kibana.yml and expect only one of your actions to be triggered.
`number_of_triggered_actions` value in event log  should be `1`.
And a warning banner on rule details page should be shown.

```
xpack.alerting.rules.execution:
  actions:
    connectorTypeOverrides:
      - id: '.server-log'
        max: 1
```
